src/components/artifacts/ArtifactsView.tsx

```diff
@@ -194,7 +194,11 @@ class ArtifactsView extends React.Component<Props> {
       <Paper elevation={1}>
         <Toolbar className={classes.title}>
           <Typography variant="h6" color="inherit">
-            {this._currentPath() || 'Artifacts'}
+            Artifacts
+          </Typography>
+          <br />
```

Comment:
There is no single `<br />` in the condebase. You should find a better way to have multiline in the `Toolbar` if you want to go this way. IMO `{this._currentPath() || 'Artifacts'}` looks nicer: if you already went down the artifacts tree you know these are the `Artifacts`. 🤷‍♂